README.md

```diff
@@ -378,6 +378,44 @@ users:
         env: null
 ```
 
+## Non-AKS cluster setup

```

Comment:
## Setup for Kubernetes OIDC Provider using Azure AD

Also, please add a link in Features above at L15